src/pybind/mgr/cephadm/nfs.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+from typing import Dict, Optional
+
+from ceph.deployment.service_spec import NFSServiceSpec
+
+import cephadm
+from orchestrator import OrchestratorError
+
+logger = logging.getLogger(__name__)
+
+class NFSGanesha(object):
+    def __init__(self,
+                 mgr,
+                 daemon_id,
+                 spec):
+        # type: (cephadm.CephadmOrchestrator, str, NFSServiceSpec) -> None
+        self.daemon_type = 'nfs'
```

Comment:
This is always going to be equal to `spec.service_type`. Is there a need to have this?